client/components/Button/Button.css

```diff
@@ -13,4 +13,8 @@
     margin-left: calc($base-spacing * 2);
     box-shadow: $button-shadow;
   }
+  &[disabled] {
+    background: #f7f7f7;
```

Comment:
were you able to check global css variables? maybe we can add, or reuse one from it.

Reply:
The grey variable is kinda specific for the stars.

$strayGray

Added a $grey variable pushing my PR